tests/lib/rules/no-tabs.js

```diff
@@ -24,10 +24,13 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n" +
         "  //   sdfdsf \n" +
         "}",
+        "var str = 'test\tstring'",
+        "var re = /test\tstring/"
```

Comment:
Oh, I see. I feel a bit bad about doing that, though. What would be the purpose of letting ESLint warn on tabs inside strings but not fixing them? Is there something we can do to fix such issues anyway?

My personal preference would be to either warn + fix everything or just ignore those occurrences which we cannot fix.